opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/BaseApi.java

```diff
@@ -472,6 +472,17 @@ public void setStringProvider(StringProvider stringProvider) {
         this.stringProvider = stringProvider;
     }
 
+    protected String buildHttpGetParams(Map<String, String> params,
+                                        String encoding) throws UnsupportedEncodingException {
```

Comment:
Still, I see that using a ``Map`` in your API implementation is a lot easier than the List of pairs (because you need to replace values multiple times).
I think it is okay like this.